From a8eb57affb305cb22ef0fe59a3a4a9dc398304a9 Mon Sep 17 00:00:00 2001
From: Yonghua Zheng <yonghua.zheng@esimtek.com>
Date: Mon, 24 Jul 2017 15:34:57 +0800
Subject: [PATCH] Patch to fix UnicodeData.h not found issue

Root cause is 'intermediates' is not properly defined
and result in target intermediate path assigned value of
/generated/UnicodeData.h and non-root user won't have
permission to create.
---
 libs/minikin/Android.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libs/minikin/Android.mk b/libs/minikin/Android.mk
index 9d82579..c13ac71 100644
--- a/libs/minikin/Android.mk
+++ b/libs/minikin/Android.mk
@@ -15,6 +15,11 @@
 LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
+
+LOCAL_MODULE := libminikin
+LOCAL_MODULE_CLASS := include
+intermediates := $(call local-generated-sources-dir)
+
 # Generate unicode emoji data from UCD.
 UNICODE_EMOJI_H_GEN_PY := $(LOCAL_PATH)/unicode_emoji_h_gen.py
 UNICODE_EMOJI_DATA := $(TOP)/external/unicode/emoji-data.txt
-- 
2.7.4

